docs/guides/extensions/hooks-validate-number-twilio.md

```diff
@@ -79,7 +79,7 @@ const accountSid = env.TWILIO_ACCOUNT_SID;
 const authToken = env.TWILIO_AUTH_TOKEN;
 const client = new twilio(accountSid, authToken);
 
-client.lookups.v2
+await client.lookups.v2
```

Comment:
Why do you think we need to `await` this call, while we already use the `then` method?